api.yml

```diff
@@ -4,8 +4,8 @@ servers:
   - description: SwaggerHub API Auto Mocking
     url: https://virtserver.swaggerhub.com/danubetech/credential-issuer/0.0.1
 info:
-  description: This is an open API specification for issuing digital credentials based on standards such as the Verifiable Credentials Data Model (https://www.w3.org/TR/vc-data-model/), JWT (https://tools.ietf.org/html/rfc7519), and Open Badges (https://openbadges.org/). Implementations of this API may differ in terms of which data formats or proof formats they do or do not support. Also, implementations may choose to not support certain parts of this API (e.g. optional parts include changing and retrieving the status of a credential, or refreshing a credential).
-  version: "0.0.1"
+  description: This is an experimental open API specification for issuing digital credentials based on standards such as the Verifiable Credentials Data Model (https://www.w3.org/TR/vc-data-model/), JWT (https://tools.ietf.org/html/rfc7519), and Open Badges (https://openbadges.org/). Implementations are expected to change significantly and implementers that are not directly involved in the work are strongly urged to not implement this API. Implementations of this API may differ in terms of which data formats or proof formats they do or do not support. Implementations may choose to not support certain parts of this API (e.g. optional parts include changing and retrieving the status of a credential, or refreshing a credential). 
+  version: "0.0.2"
   title: Credential Issuer API
   contact:
     email: markus@danubetech.com
```

Comment:
feel free to add yourself so i don't have to take all the blame alone :)

Reply:
I tried to see if it could support multiple and it didn't look like it.

Reply:
Yeah, it doesn't support multiple emails. We may just want to replace it w/ the CCG mailing list URL. I also have a plan to use ReSpec and pull the api.yml file in directly, working on that, but wanted to wait until the dust settled.

Reply:
+1 to using a ccg email in the future.

Reply:
https://github.com/w3c-ccg/vc-issuer-http-api/issues/36

Resolving, as this is now tracked seperatly.